fe4u-boilerplate-main/src/js/RawDate/userValidator.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable */
+const regular_exp = {
+    string: /^[A-Za-z]{5,10}\s[A-Za-z]{5,10}/,
+    email: /^[^\W\d_]\w+@\w+\.\w+(\.\w+)?$/,
+    phoneNumber: /(\+)?([- _():=+]?\d[- _():=+]?){8,14}/,
+}
+const validString = (str) => {
+    if(!regular_exp.string.test(str))
+        console.log("Name or second name failed");
```

Comment:
Normally you should some messages in the browser instead of `console`. Even with `alert`, if no time to show some text in DOM.